setup.py

```diff
@@ -0,0 +1,13 @@
+from setuptools import setup, find_packages
+
+VERSION = 1
+
+setup(
```

Comment:
> This is an application. Dependencies should be pinned.

That's what `requirements.txt` is for, not `setup.py`. They're not the
same thing. See link above.